dist/cleanup.js

```diff
@@ -2827,16 +2827,17 @@ module.exports = (process.env['OS'] != 'Windows_NT') ? {
 
     // Use getent() system call, since this is what ssh does; makes a difference in Docker-based
     // Action runs, where $HOME is different from the pwent
-    home: os.userInfo().homedir,
-    sshAgent: 'ssh-agent',
-    sshAdd: 'ssh-add'
+    homePath: os.userInfo().homedir,
+    sshAgentPath: 'ssh-agent',
+    sshAddPath: 'ssh-add',
+    gitPath: 'git'
 
 } : {
 
-    home: os.homedir(),
-    sshAgent: 'c://progra~1//git//usr//bin//ssh-agent.exe',
-    sshAdd: 'c://progra~1//git//usr//bin//ssh-add.exe'
-
+    homePath: os.homedir(),
+    sshAgentPath: 'c://progra~1//git//usr//bin//ssh-agent.exe',
+    sshAddPath: 'c://progra~1//git//usr//bin//ssh-add.exe',
+    gitPath: 'c://progra~1//git//usr//bin//git.exe'
```

Comment:
@ochococo Now that I take a step back, what would you think about having `homePath`, but `sshAgentCmd` (or `Command`)  and the same for other items that refer to binaries?

I can make the change for you if you agree.

Reply:
I would then argue this is not a command but rather a path to command (or executable if you will).
Bike-shedding obviously, not a strong objection from me, UTY